### ISSUE TITLE
providing any packaging 

### ISSUE THREAD

**Issue by medyagh** [lima-vm/socket_vmnet#10]
Hi I would like to know if there is a plan to create a binary or a package so the minikube users could install this ?

**Comment by jandubois** [comment]
I don't think we have any plan right now. One issue is that you have to run the daemon as `root`, so a simple homebrew package won't work.

**Comment by AkihiroSuda** [comment]
> I don't think we have any plan right now. 

I plan.

> One issue is that you have to run the daemon as `root`, so a simple homebrew package won't work.

Can we use a cask?

**Comment by jandubois** [comment]
> Can we use a cask?

That should work. I always wanted to check if you can use a cask with just a shell script, or if you need a proper `*.pkg` installer for it.

**Comment by AkihiroSuda** [comment]
PR:
- https://github.com/lima-vm/socket_vmnet/pull/11

> `*.pkg`

Seems no
https://github.com/Homebrew/homebrew-cask/blob/master/Casks/android-commandlinetools.rb

**Comment by AkihiroSuda** [comment]
Binaries are available now: https://github.com/lima-vm/socket_vmnet/releases/tag/v1.0.0

TODO: Cask (help wanted)

**Comment by medyagh** [comment]
Excellent, we look forward to use it in our documentation. would we need to uninstall the one built by source first?

**Comment by AkihiroSuda** [comment]
> would we need to uninstall the one built by source first?

That would be recommended, but actually the files can be just overwritten

**Comment by AkihiroSuda** [comment]
Now available in Homebrew
https://github.com/Homebrew/homebrew-core/commit/636ac0700e011b23c3e41cc4176b30e01a3af5ae
